fili-core/src/test/groovy/com/yahoo/bard/webservice/web/MetricParserSpec.groovy

```diff
@@ -2,26 +2,27 @@
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 package com.yahoo.bard.webservice.web
 
-import org.json.JSONArray
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.ArrayNode
 
 import spock.lang.Specification
 import spock.lang.Unroll
 
 class MetricParserSpec extends Specification {
-
-    JSONArray expectedJsonObj1
-    JSONArray expectedJsonObj2
-    JSONArray expectedJsonObj3
+    ObjectMapper mapper = new ObjectMapper()
```

Comment:
Rather than building an `ObjectMapper` directly, I would suggest going through the [ObjectMappersSuite](https://github.com/yahoo/fili/blob/master/fili-core/src/main/java/com/yahoo/bard/webservice/application/ObjectMappersSuite.java). The ObjectMappersSuite That way you're using ObjectMappers with the same configuration as we do throughout the rest of the codebase.